aptos-move/framework/aptos-framework/sources/object.move

```diff
@@ -467,21 +467,22 @@ module aptos_framework::object {
     ) acquires ObjectCore {
         let owner_address = signer::address_of(owner);
         verify_ungated_and_descendant(owner_address, object);
+        transfer_raw_inner(object, to);
+    }
 
+    inline fun transfer_raw_inner(object: address, to: address) acquires ObjectCore {
         let object_core = borrow_global_mut<ObjectCore>(object);
-        if (object_core.owner == to) {
-            return
+        if (object_core.owner != to) {
+            event::emit_event(
+                &mut object_core.transfer_events,
+                TransferEvent {
+                    object,
+                    from: object_core.owner,
+                    to,
+                },
+            );
+            object_core.owner = to;
```

Comment:
please, please no..... now everyone that is not has to read to the bottom lol, I like the rest of the PR but why this....

Reply:
Yeah less indentation previously is better

Reply:
because inline function does not support `return`...